_layouts/bib.html

```diff
@@ -2,8 +2,14 @@
 ---
 <!-- _layouts/bib.html -->
       <div class="row">
-        <div class="col-sm-2 abbr">
-        {%- if entry.abbr -%}
+        <div class="col-sm-3 {% if entry.preview %} preview{% else %} abbr{% endif %}">
```

Comment:
let's keep the column the same width
```suggestion
        <div class="col-sm-2 {% if entry.preview %}preview{% else %}abbr{% endif %}">
```